docs/UserGuide.md

```diff
@@ -166,6 +166,22 @@ Example usage of delete command in BayMeds:
 
 ![Example of delete command](images/ui/Ui-delete.png)
 
+### Listing Untaken Medication: `list today`
```

Comment:
Can help to add a space before the ':'

So it becomes Listing Untaken Medication : `list today`